backend/apps/auth/src/server.py

```
@@ -0,0 +1,36 @@
+app.add_middleware(
+    SessionMiddleware,
+    secret_key=os.getenv("SESSION_SECRET", "change-this-to-a-random-secret-key")
+)
+
+app.include_router(oauth.router, prefix="/auth/oauth", tags=["OAuth"])
+
+security = HTTPBearer()
+
+# JWT 검증
+@app.post("/auth/verify")
+def verify_token(credentials = Depends(security)):
```

Ruff (0.14.0)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)